apps/editing-toolkit/editing-toolkit-plugin/whats-new/src/whats-new.js

```diff
@@ -0,0 +1,186 @@
+			) }
+		</>
+	);
+}
+
+function getWhatsNewPages() {
+	return [
+		{
+			imgSrc: blockPatternsImage,
+			heading: __( 'New Block Patterns', 'full-site-editing' ),
+			description: createInterpolateElement(
+				/* translators: the embed is a link */
+				__(
+					'<p>Easily add patterns in the editor to save time configuring and designing.</p> <p>There are hundreds of pre-made patterns for buttons, headers, galleries, and more available via the + button inserter at the top left of all pages and posts.</p> <p><Link>Learn more about block patterns</Link>.</p>',
+					'full-site-editing'
+				),
+				{
+					Link: (
+						<a
+							href="https://wordpress.com/support/block-pattern/"
+							target="_blank"
+							rel="noreferrer"
+						/>
+					),
+					p: <p />,
+				}
+			),
+		},
+		{
+			imgSrc: paymentsImage,
+			heading: __( 'Use payments features to make money', 'full-site-editing' ),
+			description: createInterpolateElement(
+				/* translators: the embed is a link */
+				__(
+					'<p>You can process payments for just about anything on your website.</p><p>With the <Link1>Payments</Link1>, <Link2>Premium Content</Link2>, and <Link3>Donations blocks</Link3>, it takes just minutes to get set up to collect payments from your visitors. Available with any paid plan.</p><p><Link4>Get started with payments today</Link4>.</p>',
+					'full-site-editing'
+				),
+				{
+					Link1: (
+						<a
+							href="https://wordpress.com/support/wordpress-editor/blocks/payments/"
+							target="_blank"
+							rel="noreferrer"
+						/>
+					),
+					Link2: (
+						<a
+							href="https://wordpress.com/support/wordpress-editor/blocks/premium-content-block/"
+							target="_blank"
+							rel="noreferrer"
+						/>
+					),
+					Link3: (
+						<a
+							href="https://wordpress.com/support/wordpress-editor/blocks/donations/"
+							target="_blank"
+							rel="noreferrer"
+						/>
+					),
+					Link4: <a href="http://wordpress.com/earn" target="_blank" rel="noreferrer" />,
+					p: <p />,
+				}
+			),
+		},
+		{
+			imgSrc: whatsappImage,
+			heading: __( 'Let your visitors message you on WhatsApp', 'full-site-editing' ),
+			description: createInterpolateElement(
+				/* translators: the embed is a link */
+				__(
+					'<p>Connect and communicate with your website’s visitors with the new <Link>WhatsApp block</Link>.</p><p>With a single click, your visitors can ask questions or message you for whatever reason. Available with Premium, Business, and eCommerce plans.</p>',
+					'full-site-editing'
+				),
+				{
+					Link: (
+						<a
+							href="https://wordpress.com/support/wordpress-editor/blocks/whatsapp-button-block/"
+							target="_blank"
+							rel="noreferrer"
+						/>
+					),
+					p: <p />,
+				}
+			),
+		},
+	];
+}
+
+function WhatsNewPage( {
+	pageNumber,
+	isLastPage,
+	alignBottom = false,
+	heading,
+	description,
+	imgSrc,
+} ) {
+	useEffect( () => {
+		recordTracksEvent( 'calypso_editor_whats_new_slide_view', {
```

Comment:
`calypso_editor` might not be the most descriptive, since this can also be loaded in wp-admin views outside of calypso. (Unless there is already a tracks event for this :))